gitlab_api.py

```diff
@@ -94,10 +97,45 @@ def get_users(self, email_domain=None):
                 users.append(self.get_basic_user_data(user))
         return users
 
+    def project_mrs2csv(self, project_id=None):
+        if not project_id:
+            return
+
+        task_label_pattern = re.compile(r'(task|t|vx|issue|i|us|hu)[# | -]?\d+', re.IGNORECASE)
+        numbers_regex = re.compile(r'\d+')
+        task_url_tmpl = 'https://www.vauxoo.com/web#id=%s&model=project.task&view_type=form'
+
+        project = self.gl.projects.get(project_id)
+        mrs = project.mergerequests.list(state='all', scope='all', all=True)
+
+        csv_name = '%s_mrs_%s%s' % (project.name, datetime.now().strftime('%Y-%m-%d_%H-%M-%S'), '.csv')
+        with open(csv_name, 'w+', newline='') as csv_file:
+            writer = csv.writer(csv_file)
+            writer.writerow(
+                ['# de MR', 'URL del MR', 'Nombre de persona asignada', 'Título', 'Tarea', 'URL de la tarea', 'Estado',
+                 'Etiquetas', 'Fecha de creación', 'Fecha de última actualización', 'Fecha de mezcla'])
+
+            for mr in mrs:
+                task_id_found = task_label_pattern.search(mr.title)
+                task_id = ''
+                task_url = ''
+
+                if task_id_found:
+                    task_id = task_id_found[0]
+                    task_url = task_url_tmpl % numbers_regex.search(task_id)[0]
```

Comment:
if the patter i|issue matches, it means it's not a task, but an issue (ticket), isn't it?

Reply:
Yes, I think so. Also what does the other patterns mean? Like **vx | us | hu** do they use different models?

Reply:
- vx means Vauxoo issue (ticket)
- uh/hu mean user story/historia de usuario. I'd go for task, not so sure. I've never seen this being used anyway.